tools/clone_flutter.sh

```diff
@@ -17,7 +17,7 @@ fi
 cd $ENGINE_PATH/src/flutter
 
 # Special handling of release branches.
-ENGINE_BRANCH_NAME=`git branch | grep '*' | cut -d ' ' -f2`
+ENGINE_BRANCH_NAME=$CIRRUS_BASE_BRANCH
```

Comment:
Why do we need this variable? Can we just use `CIRRUS_BASE_BRANCH`?

Reply:
In summary when we wrote clone_flutter.sh script, we though git branch will fetch the corresponding branch name and we will change the branch if, test is running on a release branch. however we didn't see it in actions since all tests so far was on master. with the new release we realize git branch return (no branch). That is why web integration tests were failing for the release. so I started using this cirrus ci parameter. I tested a few more variables such as $BRANCH $CIRRUS_BRANCH only this one returns the value we need.

Reply:
What I mean is if you remove this line then search and replace ENGINE_BRANCH_NAME with CIRRUS_BASE_BRANCH, wouldn't the result be the same?

Reply:
Yes, sorry I didn't get your question! as you will see I use ENGINE_BRANCH_NAME in a lot of places. I also think is a more descriptive variable name than CIRRUS_BRANCH_NAME. We can also keep ENGINE_BRANCH_NAME when we are using this script in LUCI too.